lib/i18n/js/engine.rb

```diff
@@ -9,56 +10,58 @@ class SprocketsExtension
     end
 
     class Engine < ::Rails::Engine
-      if JS::Dependencies.sprockets_supports_register_preprocessor?
```

Comment:
If you remove this comment it might cause some errors (when sprockets lib absent)
This is why we are using `sprockets_supports_register_preprocessor?`